tests/console/gpg.pm

```diff
@@ -183,17 +182,6 @@ sub run {
     my $gpg_fips_string = check_var('FIPS_ENABLED', '1') ? "fips-mode:y::Libgcrypt" : "fips-mode:n::";
     validate_script_output("gpgconf --show-versions", sub { m/.*$gpg_fips_string.*/ }) if (is_sle('15+') && !is_jeos && !is_public_cloud);
 
-    # increase entropy for key generation for s390x on svirt backend
```

Comment:
My concern is that, can we only remove the coverage only on sle16？ :)

Reply:
You're right,  updated